blogs/scripted-windows/OOTB_install.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+OCFILE="./oc-creds.yaml"
+YAMLFILE="./scripted-windows/"
```

Comment:
This is causing troubles because we are now running from the `scripted-windows` directory.

```suggestion
YAMLFILE="."
```

Reply:
Done